src/components/NotificationsDrawer/drawerNotificationsPermissions.ts

```
@@ -0,0 +1,18 @@
+export const DRAWER_NOTIFICATIONS_V1_PERMISSIONS = [
+  'notifications:*:*',
+  'notifications:notifications:read',
+  'notifications:notifications:write',
+] as const;
+
+const isDrawerNotificationsV1Permission = (permission: string | undefined): boolean =>
+  DRAWER_NOTIFICATIONS_V1_PERMISSIONS.some((allowed) => allowed === permission);
+
+export const hasV1DrawerNotificationsPermissions = (
+  permissions: Access[] | undefined
+): boolean =>
```

ESLint

[error] Replace `⏎··permissions:·Access[]·|·undefined⏎` with `permissions:·Access[]·|·undefined`

(prettier/prettier)